.github/CODEOWNERS

```diff
@@ -133,7 +125,6 @@
 
 /.gitlab/container_scan/container_scan.yml           @DataDog/container-integrations @DataDog/agent-delivery
 
-/.gitlab/internal_image_deploy.yml                   @DataDog/container-integrations @DataDog/agent-delivery
```

Comment:
Same as above there is `.gitlab/powershell_script_deploy/powershell_script_deploy.yml` so you can probably just replace with `.gitlab/powershell_script_deploy`